feature-configs/integration-kni1/ptp/subscription.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: ptp-operator-subscription
+  namespace: openshift-ptp
+spec:
+  channel: "4.4"
+  name: ptp-operator
+  source: "opsrctest"
```

Comment:
is this subscription correct?

Reply:
I'm using the same one the demo uses here https://github.com/openshift-kni/cnf-features-deploy/blob/master/feature-configs/demo/ptp/subscription.yaml#L10

Reply:
ah, I did not notice it! Thanks for explaining